### PR TITLE
Remove self from OUFR Lists* ownership

### PR DESCRIPTION
**Pull request by KevinTCoughlin**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Removing myself as CODEOWNER bottleneck for `office-ui-fabric-react` List controls. In cases where a component owner is absent, it is likely best that @aditima or similar fill-in to guard against regression. That crew is still actively working with those components in production.

I will remain involved in ways that mutually benefit Fabric & SharePoint Framework.

#### Focus areas to test

(optional)

cc: @patmill 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12352)

### CODE REVIEW

**Comment by ecraig12345** [comment]
@dzearing Who should be listed as codeowner for list stuff instead of Kevin? I definitely don't think we should leave it ownerless.

You can list me for Check since I worked on it a little.

**Comment by dzearing** [comment]
I have put my name down for now.

**Comment by KevinTCoughlin** [comment]
> I have put my name down for now.

Thanks! @aditima may also be a candidate for guarding against regressions since her team is still primary stakeholders.

Feel free to ping me on issues / PRs as required.

**Comment by dzearing** [comment]
Asked in mail about owners

**Comment by khmakoto** [comment]
@dzearing @KevinTCoughlin what was the decision made here?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -105,7 +105,7 @@ packages/office-ui-fabric-react/src/components/FolderCover/ @ThomasMichon @bigba
 packages/office-ui-fabric-react/src/components/FocusTrapZone/ @khmakoto
 packages/office-ui-fabric-react/src/components/FocusZone/ @khmakoto
 packages/react-focus/src/components/FocusZone/ @khmakoto
-packages/office-ui-fabric-react/src/components/GroupedList/  @aditima @KevinTCoughlin
+packages/office-ui-fabric-react/src/components/GroupedList/  @aditima
```
Comment (ecraig12345) [inline .github/CODEOWNERS:108]:
```suggestion
packages/office-ui-fabric-react/src/components/GroupedList/  @aditima @dzearing
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -174,23 +174,23 @@ packages/office-ui-fabric-react/src/components/ChoiceGroup/docs/   @srideshpande
...
+packages/office-ui-fabric-react/src/components/DetailsList/docs/ @dzearing
 # packages/office-ui-fabric-react/src/components/Dialog/docs/
 packages/office-ui-fabric-react/src/components/DocumentCard/docs/ @yiminwu
 # packages/office-ui-fabric-react/src/components/Dropdown/docs/
 # packages/office-ui-fabric-react/src/components/Fabric/docs/
 # packages/office-ui-fabric-react/src/components/Facepile/docs/
 # packages/office-ui-fabric-react/src/components/FocusTrapZone/docs/
 # packages/office-ui-fabric-react/src/components/FocusZone/docs/
-packages/office-ui-fabric-react/src/components/GroupedList/docs/  @aditima @KevinTCoughlin
+packages/office-ui-fabric-react/src/components/GroupedList/docs/  @aditima
```
Comment (ecraig12345) [inline .github/CODEOWNERS:186]:
```suggestion
packages/office-ui-fabric-react/src/components/GroupedList/docs/  @aditima @dzearing
```

**Comment by khmakoto** [comment]
Also, whatever changes are done here should also be replicated in the 6.0 branch.

**Pull request merged**